src/components/dls/PopoverMenu/PopoverMenu.tsx

```diff
@@ -32,7 +32,16 @@ type PopoverMenuItemProps = {
 };
 PopoverMenu.Item = ({ children, icon, onClick, isDisabled }: PopoverMenuItemProps) => {
   return (
-    <PrimitiveDropdownMenu.Item className={styles.item} onClick={onClick} disabled={isDisabled}>
+    <PrimitiveDropdownMenu.Item
+      className={styles.item}
+      onClick={(e) => {
+        // PopoverMenu automatically close it self when one of item is clicked
+        // this code prevent that, so it only close when user click outside of the PopoverMenu
+        e.preventDefault();
```

Comment:
this should be done conditionally based on a prop. for e.g. (shouldCloseAfterClick={false})